src/main/java/com/hubspot/maven/plugins/dependency/management/DependencyManagementAnalyzer.java

```diff
@@ -48,24 +84,40 @@ private boolean checkDependencyManagement() {
         if (!projectVersion.equals(managedVersion)) {
           String errorFormat = "Version mismatch for %s, managed version %s does not match project version %s";
           log.warn(String.format(errorFormat, dependencyKey, managedVersion, projectVersion));
+          dependencyVersionMismatchError = true;
           success = false;
         } else if (originalDependency != null) {
           if (!config.allowVersions() && originalDependency.getVersion() != null) {
             log.warn(String.format("Version tag must be removed for managed dependency %s", dependencyKey));
+            dependencyVersionDisallowedError = true;
             success = false;
           }
 
           if (!config.allowExclusions() && !originalDependency.getExclusions().isEmpty()) {
             log.warn(String.format("Exclusions must be removed for managed dependency %s", dependencyKey));
+            dependencyExclusionsError = true;
             success = false;
           }
         }
       } else if (config.requireDependencyManagement()) {
         log.warn(String.format("Dependency %s is not managed", dependencyKey));
+        unmanagedDependencyError = true;
         success = false;
       }
     }
 
+    if (dependencyVersionMismatchError && requireManagement.dependencyVersionMismatchMessage() != null) {
+      errorMessages.add("Found version mismatches in managed dependencies: "+requireManagement.dependencyVersionMismatchMessage());
```

Comment:
This is a little persnickety, but can we make these messages consistent with regard to pluralization (`mismatches` vs `version` vs `exclusion`). Maybe we could make the pluralization parenthetical: `Found version mismatch(es)`, `Found version(s)`, etc.

Reply:
Sure. Should this message should look something like this?
Found version mismatch(es) in managed dependencies

Reply:
👍